support/src/main/java/se/inera/intyg/common/support/facade/util/ViewTextToolkit.java

```diff
@@ -80,10 +80,10 @@ public static String multipleStringValues(String... value) {
             return NOT_SPECIFIED;
         }
 
-        return stringBuilder.toString().trim();
+        return text;
     }
 
-    public static String withComma(String... value) {
+    public static String multipleStringValuesWithComma(String... value) {
```

Comment:
This method has "logic" that isn't covered by test. Maybe could add some test cases for handling null, trimming and empty?

Reply:
added tests!